src/components/App/index.js

```diff
@@ -446,52 +456,62 @@ class App extends React.Component {
-        />
-
-        <ModalNoData
-          filters={ this.state.filters }
-          filtersOpen ={ this.state.filtersOpen }
-          currentMode={ this.state.mode }
-          domain={ this.state.layer.domain }
-          onChangeFilters={ this.handleModal.bind(this, 'open', 'filtersOpen') }
-          onGoBack={ this.setDonationsAsmode.bind(this) }
-          onCancel={ this.resetFilters.bind(this) }
-        />
+        { !this.state.embed &&
+          <div id="map-credits" className="l-map-credits">
+            <p className="about-label text text-cta" onClick={ () => this.handleModal('open', 'aboutOpen') }>About the data</p>
+            <a className="btn-about" onClick={ () => this.handleModal('open', 'aboutOpen') }>
+              <svg className="icon icon-info">
+                <use xlinkHref="#icon-info"></use>
+              </svg>
+            </a>
+          </div>
+        }
+
+        { !this.state.embed &&
+          <ModalAbout
+            visible={ this.state.aboutOpen }
+            onClose={ this.handleModal.bind(this, 'close', 'aboutOpen') }
+          />
+        }
+
+        { !this.state.embed &&
+          <ModalFilters
+            visible={ this.state.filtersOpen }
+            onClose={ this.handleModal.bind(this, 'close', 'filtersOpen') }
+            onSave={ this.updateFilters.bind(this) }
+            wholeDomain={ layersCollection.getDataDomain() }
+            domain={ this.state.layer.domain }
+            routerParams={ this.router && this.router.params.toJSON() }
+          />
+        }
+
+        { !this.state.embed &&
+          <ModalNoData
+            filters={ this.state.filters }
+            filtersOpen ={ this.state.filtersOpen }
+            currentMode={ this.state.mode }
+            domain={ this.state.layer.domain }
+            onChangeFilters={ this.handleModal.bind(this, 'open', 'filtersOpen') }
+            onGoBack={ this.setDonationsAsmode.bind(this) }
+            onCancel={ this.resetFilters.bind(this) }
+          />
+        }
 
         <ModalDonors
           visible= { this.state.donorsOpen }
           onClose= { this.handleModal.bind(this, 'close', 'donorsOpen') }
         />
 
-        <a href="https://my.care.org/site/Donation2;jsessionid=5FED4A2DADFB975A2EDA92B59231B64B.app314a?df_id=20646&mfc_pref=T&20646.donation=form1" rel="noreferrer" target="_blank" id="donate" className="l-donate btn-contrast">
-          Donate
-        </a>
+        { !this.state.embed &&
+          <a href="https://my.care.org/site/Donation2;jsessionid=5FED4A2DADFB975A2EDA92B59231B64B.app314a?df_id=20646&mfc_pref=T&20646.donation=form1" rel="noreferrer" target="_blank" id="donate" className="l-donate btn-contrast">
+            Donate
+          </a>
+        }
       </div>;
     }
 
     return (
-      <div className={ 'l-app ' + (this.state.ready ? '' : 'is-loading') }>
+      <div className={ 'l-app ' + (this.state.ready ? '' : 'is-loading') + (this.state.embed ? 'is-embed' : '') }>
```

Comment:
Here you won't get what you want because as soon as both `this.state.ready` and `this.state.embed` are evaluated as true, you'll get the class `l-app is-loadingis-embed`.